tests/FacebookClientTest.php

```diff
@@ -218,6 +218,14 @@ public function testARequestWithFilesWillBeMultipart()
         $this->assertContains('multipart/form-data; boundary=', $headersSent['Content-Type']);
     }
 
+    public function testAFacebookRequestValidateAccessTokenIsNotProvided(){
```

Comment:
`{` should be on next line.

Also method name should be `testAFacebookRequestValidatesTheAccessTokenWhenOneIsNotProvided()` :)